snap/info.go

```diff
@@ -433,10 +437,24 @@ func (s *Info) Description() string {
 	return s.OriginalDescription
 }
 
+// Links returns the blessed set of snap-related links.
+func (s *Info) Links() map[string][]string {
+	if s.EditedLinks != nil {
+		return s.EditedLinks
+	}
+	return s.OriginalLinks
+}
+
 // Contact returns the blessed contact information for the snap.
 func (s *Info) Contact() string {
-	// TODO: consider links later
-	return s.EditedContact
+	if s.EditedContact != "" {
+		return s.EditedContact
+	}
+	contacts := s.Links()["contact"]
```

Comment:
Would it make sense to swap the order, and give more priority to the links instead of the EditedContact?

Just wondering -- I'm not really understanding what the `Edited...` fields mean.

Reply:
it means it was set in the store to be valid across the snap (instead of coming from a single revision), the naming is a bit historical, if you see in info.go we have other pairs of Edited vs Original fields, they all behave this way

Reply:
From a publisher point of view, store edited fields should have priority

Reply:
Thanks, then it makes sense the way it is :-)